tracim/development.ini.base

```diff
@@ -32,8 +32,6 @@ cookie_secret = 3283411b-1904-4554-b0e1-883863b53080
 [app:main]
 use = egg:tracim
 full_stack = true
-# You can set french as default language by uncommenting next line
-# i18n.lang = fr
 
```

Comment:
Why did you remove the example of setting a default language?

Reply:
This is the point also talked about by @buxx in #114:
`i18n.lang` is not a default but a fallback language when not using language detection from browser request header. The [documentation]( http://turbogears.readthedocs.io/en/latest/reference/config-options.html#internationalization) is quite unclear on this parameter, stating two opposite behaviors:
- "works both when language detection is enabled or disabled." followed by
- "If this is set and language detection is dislabled, the application will consider that all gettext wrapped strings must be translated to this language."

Tests shows the later prevails on the former: `i18n.lang` seems to set the language for an application not designed to be internationalized.

As we:
- have an internationalized application,
- built on language detection,
I choosed to simply drop this misleading configuration parameter.

But I may have done wrong :)

Reply:
Could you open an issue on github about the documentation and ask for confirmation of what you explain here (that i18n.fr is intended for not-internationalized applications)? https://github.com/TurboGears/tg2/issues

Then, we'll get response to the current question:
- if you're right we merge your PR as it is right now and maybe we contribute to the TG2 documentation
- if the behavior is different, then we'll see what alessandro molina explain to us.
-

Reply:
Given the answer received from @amol-, I was wrong:
https://github.com/TurboGears/tg2docs/issues/42
So i'm going to not only insert back this parameter but uncomment it for good and set it to English.